app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :post
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
+end
```

Comment:
- [ ]  Based on ERD diagram provided Like belong to the  post as you did,  they should belong to the user who created them as well.  Please add the association as well for your app to work properly.